ui/src/routes/_layout/create-organization.tsx

```diff
@@ -139,5 +142,12 @@ const CreateOrganizationPage = () => {
 };
 
 export const Route = createFileRoute('/_layout/create-organization')({
-  component: CreateOrganizationPage,
+  component: function Layout() {
+    const user = useUser();
+    return (
+      <PageLayout sections={rootSections(user.hasRole(['superuser']))}>
+        <CreateOrganizationPage />
+      </PageLayout>
```

Comment:
Generally this sort of layout sharing should be done by creating a `_adminSidebarLayout` etc. directory in the current `_layout` directory, add a `route.tsx` there with the shared PageLayout with the same sections and move `create-organization.tsx` and `index.tsx` there (or handling it with file names like `_adminSidebarLayout.route.tsx`, `_adminSidebarLayout.index.tsx` and `_adminSidebarLayout.create-organization.tsx` directly in _layout directory). That ensures that all shared layouts are handled the same way, so no need to go through and process what `rootSections()` etc. do.

Here I however wonder whether we could and should just omit the sidebar sections from create-organization page? I think with the current amount of links in the sidebar, it's not really important to be able to select from those in there. As `runs/index` is also available only for superusers, should it maybe be under the admn route and have a link to it there?

Reply:
Ah ok, I understood it wrong then. Well, the thing is, if you omit it from the create organization page, then there will be this jump in the layout when accessing that page, so that's mainly why I would add it there.

>As runs/index is also available only for superusers, should it maybe be under the admin route and have a link to it there?

So to move it to the admin dashboard? I figured this might be something that would be good to be more easily accessible from the front page, but I'm not sure. Or did you mean just change the link?

Reply:
> Ah ok, I understood it wrong then. Well, the thing is, if you omit it from the create organization page, then there will be this jump in the layout when accessing that page, so that's mainly why I would add it there.
> 
> > As runs/index is also available only for superusers, should it maybe be under the admin route and have a link to it there?
> 
> So to move it to the admin dashboard? I figured this might be something that would be good to be more easily accessible from the front page, but I'm not sure. Or did you mean just change the link?

Yeah exactly, move it to the admin dashboard, probably as a sub-page there. I think as it's a superuser functionality it doesn't really need to be that easily accessible from the front page.

Some layout jumping is totally fine if the pages have different layout. I think it also maybe doesn't make sense to show the sidebar on the front-page, as normal users see only one item there for the page that they're already on, so it provides no value for the regular user.

Reply:
> Yeah exactly, move it to the admin dashboard, probably as a sub-page there. 

Alright, I guess I'll change this to a draft for a bit and do some refactoring on the admin dashboard layout first to align it with the rest of the ui before adding this there.